src/Decryptor.ts

```diff
@@ -0,0 +1,5 @@
+export default interface Decryptor {
```

Comment:
This is the equivalent of https://github.com/readium/web/blob/028dfff7b9a0b34f529504cd039f0bac411cb045/shared-models/src/Publication/Encryption/Encryption.ts 

which makes me think that it does belong here, but I need to do another pass on the actual methods to make sure it all makes sense.  Ignore this file for now, I think it goes best with the PR in `axisnow-access-control-web`